tracee-ebpf/tracee/net_events.go

```diff
@@ -63,17 +62,43 @@ func (t *Tracee) processNetEvents() {
 						t.handleError(err)
 						continue
 					}
+					networkProcess, err := t.getProcessCtx(int(hostTid))
+					hr, min, sec := timeStampObj.Clock()
+					nsec := uint16(timeStampObj.Nanosecond())
+
+					if err != nil {
+						fmt.Printf("%v:%v:%v:%v  %-16s  %-7d  debug_net/packet               Len: %d, SrcIP: %v, SrcPort: %d, DestIP: %v, DestPort: %d, Protocol: %d\n",
+							hr,
+							min,
+							sec,
+							nsec,
+							comm,
+							hostTid,
+							pktLen,
+							netaddr.IPFrom16(pktMeta.SrcIP),
+							pktMeta.SrcPort,
+							netaddr.IPFrom16(pktMeta.DestIP),
+							pktMeta.DestPort,
+							pktMeta.Protocol)
+					} else {
+						fmt.Printf("%v:%v:%v:%v   %v    %-16s  %v  %v    %d             debug_net/packet              Len: %d, SrcIP: %v, SrcPort: %d, DestIP: %v, DestPort: %d, Protocol: %d\n",
+							hr,
+							min,
+							sec,
+							nsec,
+							networkProcess.Uid,
+							comm,
+							networkProcess.Pid,
+							networkProcess.Tid,
+							0,
+							pktLen,
+							netaddr.IPFrom16(pktMeta.SrcIP),
+							pktMeta.SrcPort,
+							netaddr.IPFrom16(pktMeta.DestIP),
+							pktMeta.DestPort,
+							pktMeta.Protocol)
```

Comment:
yes sure!
i made another PR that will add it .
the reason for 2 different PRs is because this one only adding the process context map management.
the second PR will take some data from the process context map and will create a external.Event struct and send it to the events channel